src/TicketMachine.java

```diff
@@ -1,72 +1,101 @@
+import org.jetbrains.annotations.NotNull;
+
 import java.util.Scanner;
 import java.util.ArrayList;
 
 public class TicketMachine {
 
-    TicketList TL1;
-    ArrayList<Ticket> temporaryReservedTicketsList = new ArrayList<>();
-    Ticket ticketChosen;
-    double balance;
-    String currency = "$";
+    private ArrayList<Ticket> availableTicketsList = new ArrayList<>();
+    private ArrayList<Ticket> temporarilyReservedTicketsList = new ArrayList<>();
+    private Ticket ticketChosen;
+    private double balance;
+    private String currency = "$";
+
 
     TicketMachine() {
-        TL1 = new TicketList();
+        Ticket SW4 = new Ticket("StarWars 4", 12);
```

Comment:
The convention is that variables start with a lowercase letter

Reply:
So an object is always a variable? was the sentence I was about to type when I realised, Yes, you idiot. An object is just a variable of the class type. So Ticket SW4 is a variable of the Ticket type... Duh
Thank you for the realisation :)